src/shared/components/custom-toast/CustomToast.tsx

```
@@ -0,0 +1,18 @@
+
+
+interface CustomToastProps extends ToastContentProps {
+    text?: string,
+    icon?: React.ReactNode,
+    children?: React.ReactNode,
+}
+
+export default function CustomToast({text, icon, children}: CustomToastProps) {
+  return (
+    <div  className="flex items-center gap-[1rem] px-[1.6rem] w-[33.5rem] h-[4rem] py-[0.9rem] bg-grayscale-900 text-white rounded-[1.6rem]">
+        {icon && icon}
```

ESLint

[error] Expected indentation of 6 spaces but found 8.

(indent)